server/server_test.go

```diff
@@ -559,6 +559,34 @@ var _ = Describe("Server package test", func() {
 					testURL("PUT", url, memberTokenID, testResource, http.StatusUnauthorized)
 				})
 			})
+
+			Context("Creating resource in different tenant", func() {
```

Comment:
What do you think about add tests for checking case when related resource has no tenant assigned. I mean the resources which are tenant or domain independent

Reply:
that's good idea, will add that